Python/Product/Analysis/Interpreter/Pep440Version.cs

```diff
@@ -20,10 +20,9 @@
 using System.Linq;
 using System.Text;
 using System.Text.RegularExpressions;
-using System.Threading.Tasks;
 
-namespace Microsoft.PythonTools.EnvironmentsList {
-    struct Pep440Version : IComparable<Pep440Version>, IEquatable<Pep440Version> {
+namespace Microsoft.PythonTools.Interpreter {
+    public struct Pep440Version : IComparable<Pep440Version>, IEquatable<Pep440Version> {
```

Comment:
>Pep440Version [](start = 18, length = 13)

can we just call this PythonVersionSpec or something a little more friendly?  We can always put in the comment that it's a PEP 440 compatible version...  I just don't see too many people seeking out "Pep..." as type names ;)